### DISCUSSION TITLE
With a Single RTX 4090 system, Core i9-13990k CPU and 96GB of RAM (Win 11), are these the optimal defaults to use (in general)?

### DISCUSSION THREAD

**Discussion by cleverestx** [CarlGao4/Demucs-Gui#46] [Q&A] [answered]
![image](https://github.com/CarlGao4/Demucs-Gui/assets/28905253/72387846-8aff-4e79-bdd4-112ed1bdbcda)

**Comment by CarlGao4** [comment, marked as answer]
No. The default parameters are calculated this way:
Segment is defined by model; overlap is 0.25 and shift is 0. If you increase shifts, the results can become better (by separating it more times and calculating the average) but take longer time. The actual performance depends on your audio file and you may try different values to get better results 😄
